utils/src/treev2/quickPickWizard/quickPickAzureResource/tempTypes.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApplicationResource, ApplicationSubscription } from '../../../../hostapi.v2';
+import * as types from '../../../../index';
+
+// TODO: THIS FILE IS TEMPORARY //
+// It needs to be replaced by real Resources extension interfaces //
+
+// These are assumptions made about the nodes in the tree

```

Comment:
Is there a way we could formalize this in the V2 API definitions without necessarily having them be blind casts?  If we strongly control the TDP functions we expose, we could do something like expose the `ResourceGroupsItem` type with a `type` property of `'Subscription' | 'Grouping' | 'Resource' | 'BranchData'` or some such thing.